### ISSUE TITLE
No CSS styling for <aside>

### ISSUE THREAD

**Issue by DataKinds** [Kimeiga/bahunya#5]
Hello! First off, thank you so much for this CSS framework -- it makes my life so much easier.

However, one thing I noticed is that there's no explicit styling for `aside` and `section > aside`. While this isn't a killer feature or anything, it would be very nice to have callout boxes that flow beautifully like the rest of the elements in this stylesheet.

MDN link: [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside).

Would be happy to make a PR if you have any thoughts as to how it should be styled.

**Comment by Kimeiga** [comment]
HEY! I apologize for the lack of communication but I'm basically almost done revamping the entire framework to be lighter, have a more internally consistent codebase, and basically look a lot better. When I release i'll let you know.

I've styled the aside with a float right and a smaller width but when you see it let me know if you want something else.

I will probably finish this after finals week so basically this summer so sorry for the delay!

**Comment by DataKinds** [comment]
Heck yeah! Take as long as you need. Looking forward to seeing it.

**Comment by Kimeiga** [comment]
for sure haha; school first i guess hahaha

**Comment by Kimeiga** [comment]
Hi Tyler! 

It's been forever but I finally got around to reworking the whole framework.

I added some styling for `<aside>`! 
<img width="793" alt="image" src="https://user-images.githubusercontent.com/20587215/173982183-fa58000d-7d1e-4e32-b373-2a31c5f2fa01.png">

Let me know if you want something special for `<section> <aside>`.

You'll have to drop in the new CDN link as I have changed the folder structure quite a lot.

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/kimeiga/bahunya/dist/bahunya.min.css">
```

Cheers and thanks for making an issue!
